### ISSUE TITLE
Cannot choose any value in the User Group list when creating new user

### ISSUE THREAD

**Issue by tlammer** [specify/specify7#396]
When creating a new user, the default is Manager and it cannot be changed.

**Comment by tlammer** [comment]
Upon further testing, you can change the value but only after the record has been saved.

**Comment by maxxxxxdlp** [comment]
Superseded by #441
